_beats/libbeat/testing/available_port.go

```diff
@@ -0,0 +1,38 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
should we exclude `_beats/libbeat/testing` ?

Reply:
Afaik the testing package gives support around the [`./apm-server test`](https://www.elastic.co/guide/en/apm/server/current/command-line-options.html#test-command) command.